lib/simplifyExpression/basicsSearch/removeAdditionOfZero.js

```diff
@@ -1,30 +1,30 @@
-const clone = require('../../util/clone');
+const {rewriteNode} = require('math-rules');
 
 const ChangeTypes = require('../../ChangeTypes');
+const defineRuleString = require('../../util/defineRuleString');
 const Node = require('../../node');
 
+// e.g. 2 + 0 -> 2
+const REMOVE_ADDING_ZERO = defineRuleString('#a + 0', '#a');
+
+// e.g. 0 + 2 -> 2
+const REMOVE_ADDING_ZERO_REVERSE = defineRuleString('0 + #a', '#a');
+
```

Comment:
what happens if 0 is in the middle? or do you need to do this to match the end cases - would be worth adding a comment to explain why you have both